inscriptions/ISic000522.xml

```diff
@@ -190,18 +195,30 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#GC">
+                <ab>
+                    <lb n="1"/><expan><abbr>D</abbr><ex>is</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>m</abbr><ex>anibus</ex></expan>
+                    <lb n="2"/><name>Aeliae</name> <g ref="#interpunct">·</g> <name>Hilare</name>
+                    <lb n="3"/>quae <g ref="#interpunct">·</g> vixit <g ref="#interpunct">·</g> <expan><abbr>an</abbr><ex>nis</ex></expan>
+                    <lb n="4"/><num value="27">XXVII</num> <g ref="#interpunct">·</g> <expan><abbr>m</abbr><ex>ensibus</ex></expan> <g ref="#interpunct">·</g> <num value="9">VIIII</num>
+                    <lb n="5"/>dieibus <g ref="#interpunct">·</g> <num value="4">IIII</num>
```

Comment:
NB the Latin has 'dies', which is complete and the accusative, which in turn means that 'an' should be an(nos) and m should be m(enses).